### PR TITLE
Section How to contribute

### PR DESCRIPTION
**Pull request by danielsous**
Fixes #

## Proposed Changes

1. ... Improved how to contribute section

### Author/Contributor
- [ ] If documentation is needed for this change, has that been included in this pull request
- [ ] Remember to check if code coverage decrease, if so, please implement the necessary tests

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [x] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

File: docs/project/contributing.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Glad you are here! Pull requests, or feature requests, though not always impleme
 Considering that the ecosystem of Herbs has numerous libraries, each one has its own specificity that you will find in this documentation in a more detailed way. 
 However, if you want to submit a fix or feature, take a look at the [CONTRIBUTING](https://github.com/herbsjs/herbs/blob/main/.github/CONTRIBUTING.md) readme in the Github and go ahead and open a ticket.
 
+## Open Development
```
Comment (italojs) [inline docs/project/contributing.md:13]: instead `Open Development`, we could to use `getting started`

File: docs/project/contributing.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ Glad you are here! Pull requests, or feature requests, though not always impleme
...
+All work on Herbs happens directly on GitHub. Both core team members and external contributors send pull requests which go through the same review process.
+
+
 By default **Herbs organization** uses [Angular Commit Message Conventions](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-format).
```
Comment (italojs) [inline docs/project/contributing.md:17]: It must to be explained in commit session

File: docs/project/contributing.md [hunk 3]
```diff
@@ -10,6 +10,10 @@ Glad you are here! Pull requests, or feature requests, though not always impleme
...
 
 Tools such as [commitizen](https://github.com/commitizen/cz-cli) or [commitlint](https://github.com/conventional-changelog/commitlint) can be used to help contributors and enforce valid commit messages.
```
Comment (italojs) [inline docs/project/contributing.md:19]: It must to be explained in commit session

File: docs/project/contributing.md [hunk 4]
```diff
@@ -26,8 +30,32 @@ To help with this task we have implemented the commitizen to all libraries, just
 - Push to your fork and submit a pull request
```
Comment (italojs) [inline docs/project/contributing.md:30]: I think we could to slit it into sessions
## how to "find" a issue
- explain what is a issue
- where is the issues
- < gif >
### how to open a issue
- < gif > 

## "downloading" and running the code
- how to fork, why fork...
- create branch
- run tests
< gif >

## commting and poushing
- how to use commitizen
- < gif >

# PR
- what is a PR
- how to open a great PR
- < gif >


##

File: docs/project/contributing.md [hunk 5]
```diff
@@ -26,8 +30,32 @@ To help with this task we have implemented the commitizen to all libraries, just
...
 - Pat yourself on the back and wait for your pull request to be reviewed and merged.
 
+## Pull Request
+First of all, we leave two topics on the [Resources](#resources) tab, explaining how to perform your first pull request and among other details about Github.
+
+After that, you need to identify that your pain can be a bug, an implementation or just a question, so it is necessary to always check the issues that are open for the project you are using and want to work on.
```
Comment (italojs) [inline docs/project/contributing.md:36]: I think this explanation about PR is confuse

File: docs/project/contributing.md [hunk 6]
```diff
@@ -26,8 +30,32 @@ To help with this task we have implemented the commitizen to all libraries, just
...
+
+If you decide to fix an issue, please be sure to check the comment thread in case somebody is already working on a fix. If nobody is working on it at the moment, please leave a comment stating that you intend to work on it so other people don’t accidentally duplicate your effort.
+
+If somebody claims an issue but doesn’t follow up for more than two weeks, it’s fine to take it over but you should still leave a comment.
+
+## Community Contributions
```
Comment (italojs) [inline docs/project/contributing.md:42]: I think we could to change to "no-code contributions"

File: docs/project/contributing.md [hunk 7]
```diff
@@ -26,8 +30,32 @@ To help with this task we have implemented the commitizen to all libraries, just
...
+Contributions to Herbs are not limited to GitHub. You can help others by sharing your experience using Herbs, whether that is through blog posts, presenting talks at conferences, or simply sharing your thoughts on Twitter and tagging @herbsjs. We also have a server in [Discord](https://discord.com/invite/e3cQ66KDv5) to focus our doubts and chat!
+
+## Why do you want to contribute to open source?
+We have several reasons to contribute to open source projects but we can summarize the main and most important ones.
+- Build your confidence as a developer:
```
Comment (italojs) [inline docs/project/contributing.md:47]: instead use topics, we could use titles

**Review by italojs — CHANGES_REQUESTED** [review]

**Review by italojs — CHANGES_REQUESTED** [review]

File: docs/project/contributing.md [hunk 8]
```diff
@@ -10,24 +10,140 @@ Glad you are here! Pull requests, or feature requests, though not always impleme
...
+```
+`Specific file, example file called usecase.js`
+```sh
+git add usecase.js
+```
+
+3. Now is the time for us to write a summary of what we've worked on. We will use the command below to initiate the commit message.
+
+```sh
+git cz
```
[the remark this answers is not in the document]
Comment (danielsous) [reply docs/project/contributing.md]: Done

File: docs/project/contributing.md [hunk 9]
```diff
@@ -10,24 +10,140 @@ Glad you are here! Pull requests, or feature requests, though not always impleme
...
+## Clone
+After Fork, you will automatically be inside the cloned repository, if not, just click on your photo and go to your repositories. There you will be able to locate the cloned repository.
+
+Now we need to download it to your development environment so you can code.
+
+1. Access the cloned repository, in our example it is buchu.
+
+2. Now just click on the green button called Code.
+
+3. Some download forms like .zip will be displayed, but in our case we will use HTTPS.
```
[the remark this answers is not in the document]
Comment (danielsous) [reply docs/project/contributing.md:57]: Done

**Comment by italojsv** [comment]
could you include this labels?
https://github.com/herbsjs/herbs/labels

**Comment by jhomarolo** [comment]
Also, please include those links:

Open issues ready to work and without an owner yet: https://github.com/search?o=desc&q=org%3Aherbsjs+label%3Aready-to-work+-label%3Awip+state%3Aopen&s=created&type=Issues

Open issues with work in progress: https://github.com/search?q=org%3Aherbsjs+-label%3Aready-to-work+label%3Awip+state%3Aopen&type=Issues 

Open issued with help wanted or extra attention needed: https://github.com/search?o=desc&q=org%3Aherbsjs+label%3A%22help+wanted%22+state%3Aopen&s=created&type=Issues

**Review by italojsv — APPROVED** [review]

**Review by italojs — APPROVED** [review]

File: docs/project/contributing.md [hunk 10]
```diff
@@ -28,6 +28,24 @@ Tools such as [commitizen](https://github.com/commitizen/cz-cli) or [commitlint]
 ## Issues
 First step to be done is access the project you want to contribute to and take a look at the issue tab, maybe there is already an issue open you want to work on, or you can even open one to be discussed.
 
+**Labels** are automatically defined when the issue/PR is opened and are updated if needed by core members.
```
Comment (jhomarolo) [inline docs/project/contributing.md:31]: @danielsous labels are not automatically defined, just the type of issue label is automatically fixed 

Always an org member tag those issues

File: docs/project/contributing.md [hunk 11]
```diff
@@ -28,6 +28,24 @@ Tools such as [commitizen](https://github.com/commitizen/cz-cli) or [commitlint]
...
+
+`wip` -> Working In Progress.
+
+`bug` -> Something isn't working.
+
+To know other tags and their reasons, access the link below:
+
+[Herbs Labels](https://github.com/herbsjs/herbs/labels)
+
 We will use the [buchu](https://github.com/herbsjs/buchu) project as an example.
```
Comment (jhomarolo) [inline docs/project/contributing.md:49]: please use herbs as example

Comment (jhomarolo) [reply docs/project/contributing.md:49]: https://github.com/herbsjs/herbs/

File: docs/project/contributing.md [hunk 12]
```diff
@@ -36,9 +54,17 @@ https://github.com/herbsjs
 ```
 2. Click on the Issues tab
 
-Just choose which question you want to act on and comment that you will act so other members are aware and your effort is not wasted.
+Just choose which issue you want to act on and comment that you will act so other members are aware and your effort is not wasted.
 You will have access to all open issues and also those that have already been closed.
 
+Below we have some links to make your search easier if you want to know all the open PRs without anyone acting or want to give tips or even help someone in an issue that is under development.
+
```
Comment (jhomarolo) [inline docs/project/contributing.md:61]: Please add a Header before this paragraph, like: "Want to know how to start?" or something simular

**Review by jhomarolo — CHANGES_REQUESTED** [review]

**Pull request merged**